frontend/pages/404.tsx

```diff
@@ -0,0 +1,22 @@
+import Layout from "@components/templates/Layout";
+import React from "react";
+import { Heading, Text } from "@chakra-ui/react";
```

Comment:
![bitmoji](https://sdk.bitmoji.com/render/panel/c76f71df-ac5b-43f4-8419-7b461d68d0f2-cbc08238-ab4d-4b30-bbee-619cefc93217-v1.png?transparent=1&palette=1&width=246)

>Text is defined, but never used

```suggestion
import { Heading } from "@chakra-ui/react";
```